Mappers' Guild September Newspost (#2602)
ppy/osu-wiki: news/2019-09-27-mappers-guild-september-newspost.md

- [Cranky - Conflict](https://osu.ppy.sh/beatmapsets/936126) by [Icekalt](https://osu.ppy.sh/users/5410645)
- [Cranky - Amanojaku](https://osu.ppy.sh/beatmapsets/978694) by [Mao](https://osu.ppy.sh/users/2204515)
- [J. Pachelbel - Canon (Cranky Remix)](https://osu.ppy.sh/beatmapsets/978674) by [Nao Tomori](https://osu.ppy.sh/users/5364763)
- [L. V. Beethoven - Moonlight Sonata (Cranky Remix)](https://osu.ppy.sh/beatmapsets/978759) by [Nwolf](https://osu.ppy.sh/users/1910766)
- [Cranky - EMPEROR](https://osu.ppy.sh/beatmapsets/979438) by [JBHyperion](https://osu.ppy.sh/users/4879508)
- [Cranky - JULIAN](https://osu.ppy.sh/beatmapsets/1008310) by [ScubDomino](https://osu.ppy.sh/users/8972308)

![The Flashbulb header image](https://assets.ppy.sh/artists/37/header.jpg)

> The **Flashbulb Standard Beatmap Pack** quest got completed by **[Pachiru](https://osu.ppy.sh/users/2850983)**, **[Gabe](https://osu.ppy.sh/users/654108)**, **[Zetera](https://osu.ppy.sh/users/587737)**, **[polka](https://osu.ppy.sh/users/4191702)**, **[greenhue](https://osu.ppy.sh/users/4967662)**, **[Irreversible](https://osu.ppy.sh/users/1287964)**, **[MBomb](https://osu.ppy.sh/users/3071175)**, **[Mismagius](https://osu.ppy.sh/users/19048)**, **[Leader](https://osu.ppy.sh/users/631530)**, **[fieryrage](https://osu.ppy.sh/users/3533958)** and **[Deif](https://osu.ppy.sh/users/318565)**.

Reviewer: Shouldn't it be **The Flashbulb** (w/ "the" bolded) instead of  The **Flashbulb** tho?

(i'm quite sure that The Flashbulb has never been known or referred to as only "Flashbulb" btw)

The **Neuro Genre Showcase** quest got completed by **[Nifty](https://osu.ppy.sh/users/4956097)**, **[Cynplytholowazy](https://osu.ppy.sh/users/3901754)**, **[Faputa](https://osu.ppy.sh/users/845733)**, **[\_DUSK\_](https://osu.ppy.sh/users/6092181)**, **[Yales](https://osu.ppy.sh/users/2377881)**, **[Nepuri](https://osu.ppy.sh/users/6637817)**, **[Tyistiana](https://osu.ppy.sh/users/1421452)** and **[aabc271](https://osu.ppy.sh/users/155707)**

- [sakuraburst - cherry blossoms explode across the dying horizon](https://osu.ppy.sh/beatmapsets/878621) by [Cynplytholowazy](https://osu.ppy.sh/users/3901754)
- [Dictate - Cauldron](https://osu.ppy.sh/beatmapsets/942334) by [Nepuri](https://osu.ppy.sh/users/6637817)
- [sakuraburst - dragonlands](https://osu.ppy.sh/beatmapsets/935765) by [Faputa](https://osu.ppy.sh/users/845733)
- [Culprate & Joe Ford - Gaucho](https://osu.ppy.sh/beatmapsets/936234) by [Nifty](https://osu.ppy.sh/users/4956097)
- [sakuraburst - SELF DESTRUCT](https://osu.ppy.sh/beatmapsets/963193) by [\_DUSK\_](https://osu.ppy.sh/users/6092181)

![tieff header image](https://assets.ppy.sh/artists/34/header.jpg)

> The **tieff Mini-pack** quest got completed by **[Mirash](https://osu.ppy.sh/users/2841009)**, **[schoolboy](https://osu.ppy.sh/users/8722791)**, **[Hinsvar](https://osu.ppy.sh/users/1249323)**, **[Realazy](https://osu.ppy.sh/users/918297)**, **[PandaHero](https://osu.ppy.sh/users/1233255)**, **[Trynna](https://osu.ppy.sh/users/2652951)**, **[eiri-](https://osu.ppy.sh/users/3388410)** and **[Delis](https://osu.ppy.sh/users/1603923)**.

Reviewer: Delis changed their name a few days ago into ShirohaMyMommy btw : http://osu.ppy.sh/users/1603923

(i thought it would be better to change it to keep stuffs consistent since you had "type 1 if cute" instead of "bor" in Line 54 already)

- [Ben Briggs - Fearless Living](https://osu.ppy.sh/beatmapsets/934415) by [Lasse](https://osu.ppy.sh/users/896613)
- [Ben Briggs - Yes (Maybe)](https://osu.ppy.sh/beatmapsets/933940) by [Myxo](https://osu.ppy.sh/users/2202645)
- [Ben Briggs - New Game Plus](https://osu.ppy.sh/beatmapsets/934627) by [Icekalt](https://osu.ppy.sh/users/5410645)
- [Ben Briggs - Wave Island](https://osu.ppy.sh/beatmapsets/934666) by [Gero](https://osu.ppy.sh/users/1467715)
- [Ben Briggs - Methods](https://osu.ppy.sh/beatmapsets/948165) by [LMT](https://osu.ppy.sh/users/7262798)
- [Ben Briggs - Selecting Continue](https://osu.ppy.sh/beatmapsets/948868) by [Mao](https://osu.ppy.sh/users/2204515)
- [Ben Briggs - Why Are We Yelling](https://osu.ppy.sh/beatmapsets/952380) by [Crissa](https://osu.ppy.sh/users/5405836)

---

Please note that there is few other finished quests, but since they include less than 5 mapsets, we decided to not having them here, to avoid having a bloated newspost.

> In the future, we plan to have a shorter and more cleanly designed newspost. This post was initially made to introduce the concept and also to keep people informed about the progess of the Mappers' Guild since the [last newspost](https://osu.ppy.sh/home/news/2018-03-17-new-mappers-guild-pack-release) about it.

Reviewer 2: remove (prev suggestion makes this unnecessary)

> See you next time!

Reviewer 2: ```suggestion
Until then, enjoy playing all of these maps!
```
... or something like that. to fit better with prev suggestion

> —Pachiru, clayton

Reviewer 2: ```suggestion
—Pachiru
```
don't think I wrote enough of this post to count as an author

---
layout: post
title: Mappers' Guild Quests Completed
date: 2019-09-24 00:00:00 +0000
---

The Mappers' Guild has been hard at work creating beatmaps for your favourite Featured Artist tracks. If you haven't been following, come check them out!

![](/wiki/shared/news/banners/mappersguild.png)

> Since March 2019, the **[Mappers' Guild](https://mappersguild.com)** has been open to everyone having at least three ranked maps. This has made the Guild very productive in terms of mapping and modding. With the introduction of the quest system, mappers have been more active and interested in mapping Featured Artist content than ever before.

Reviewer 3: ```suggestion
Since March 2019, the **[Mappers' Guild](https://mappersguild.com)** has been open to everyone with at least three ranked maps. This has made the Guild very productive in terms of mapping and modding. With the introduction of the quest system, mappers have been more active and interested in mapping Featured Artist content than ever before.
```

> The quests are usually artist-specific, but can sometimes be thematic, such as Vocaloid-themed quests for instance, which mixes several artists in one quest. They usually requires mappers to produce a given amount of maps, with sometimes extra rules that make quests more diverse. 

Reviewer 3: ```suggestion
The quests are usually artist-specific, but can sometimes be targeted in nature, such as Vocaloid-themed quests for example, which mixes several artists in one quest. They usually requires mappers to produce a given amount of maps, with sometimes extra rules that make quests more diverse. 
```

If you're curious and want to know more about the Guild and its quest system, please visit [the official Mappers' Guild website](https://mappersguild.com)!

There are a lot of completed quests, so here we've picked out the ones involving the most mapsets and mappers. You can find even more Featured Artist content by searching the [beatmap listing](https://osu.ppy.sh/beatmapsets?q=featured%20artist)!

> ![Cranky header image](https://assets.ppy.sh/artists/23/header.jpg)

Reviewer 3: ```suggestion
![](https://assets.ppy.sh/artists/23/header.jpg)
```
No need to add a subtitle for hard image links. Not sure how the news parser handles them.

Reviewer 2: these aren't titles, they're alt text. works the same on news as wiki (except for the first header )

- [Cranky - Conflict](https://osu.ppy.sh/beatmapsets/936126) by [Icekalt](https://osu.ppy.sh/users/5410645)
- [Cranky - Amanojaku](https://osu.ppy.sh/beatmapsets/978694) by [Mao](https://osu.ppy.sh/users/2204515)
- [J. Pachelbel - Canon (Cranky Remix)](https://osu.ppy.sh/beatmapsets/978674) by [Nao Tomori](https://osu.ppy.sh/users/5364763)
- [L. V. Beethoven - Moonlight Sonata (Cranky Remix)](https://osu.ppy.sh/beatmapsets/978759) by [Nwolf](https://osu.ppy.sh/users/1910766)
- [Cranky - EMPEROR](https://osu.ppy.sh/beatmapsets/979438) by [JBHyperion](https://osu.ppy.sh/users/4879508)
- [Cranky - JULIAN](https://osu.ppy.sh/beatmapsets/1008310) by [ScubDomino](https://osu.ppy.sh/users/8972308)

> ![The Flashbulb header image](https://assets.ppy.sh/artists/37/header.jpg)

Reviewer 3: ```suggestion
![](https://assets.ppy.sh/artists/37/header.jpg)
```

For this quest, the mappers had to **create and rank at least 6 mapsets of songs by The Flashbulb, each hosted by a different user**.

> **The Flashbulb Standard Beatmap Pack** quest was completed by **[Pachiru](https://osu.ppy.sh/users/2850983)**, **[Gabe](https://osu.ppy.sh/users/654108)**, **[Zetera](https://osu.ppy.sh/users/587737)**, **[polka](https://osu.ppy.sh/users/4191702)**, **[greenhue](https://osu.ppy.sh/users/4967662)**, **[Irreversible](https://osu.ppy.sh/users/1287964)**, **[MBomb](https://osu.ppy.sh/users/3071175)**, **[Mismagius](https://osu.ppy.sh/users/19048)**, **[Leader](https://osu.ppy.sh/users/631530)**, **[fieryrage](https://osu.ppy.sh/users/3533958)** and **[Deif](https://osu.ppy.sh/users/318565)**.

Reviewer 4: misleading to call it just "standard" when 1 of the 6 mapsets was osu!catch and 2 of the 10 members (MBomb, Deif) were in the quest to do that set.

Author: I named it after the name of the quest on the mappers' guild website, I can eventually change this to hybrid if you think it's better

Reviewer 4: the one on the website is just called `The Flashbulb Beatmap Pack` with the taiko one being `The Flashbulb Taiko Beatmap Pack` 

so I think hybrid would be better, it already doesn't match the mpg website listing, but would be more descriptive

- [The Flashbulb - DIDJ PVC](https://osu.ppy.sh/beatmapsets/965178) by [tasuke912](https://osu.ppy.sh/users/2774767)
- [The Flashbulb - Creep](https://osu.ppy.sh/beatmapsets/966087) by [Greenshell](https://osu.ppy.sh/users/8693851)
- [The Flashbulb - Six Acid Strings](https://osu.ppy.sh/beatmapsets/966451) by [Nardoxyribonucleic](https://osu.ppy.sh/users/876419)
- [The Flashbulb - Amen Iraq](https://osu.ppy.sh/beatmapsets/966277) by [Volta](https://osu.ppy.sh/users/4154071)
- [The Flashbulb - Black Lawn Finale](https://osu.ppy.sh/beatmapsets/967099) by [Nofool](https://osu.ppy.sh/users/672430)
- [The Flashbulb - Ninedump](https://osu.ppy.sh/beatmapsets/984361) by [Arrival](https://osu.ppy.sh/users/1694000)

> ![Rin / Function Phantom header image](https://assets.ppy.sh/artists/14/header.jpg)

Reviewer 3: ```suggestion
![](https://assets.ppy.sh/artists/14/header.jpg)
```

- [Rin - Amanojaku set 03 ~ Romantic Escape Flight](https://osu.ppy.sh/beatmapsets/1022358) by [Halfslashed](https://osu.ppy.sh/users/4598899)
- [Rin - Amanojaku set 02 ~ Midnight Spell Card](https://osu.ppy.sh/beatmapsets/974872) by [Aeril](https://osu.ppy.sh/users/4334976)
- [Rin - Eientewi set 14 Another](https://osu.ppy.sh/beatmapsets/1022069) by [Mentai](https://osu.ppy.sh/users/8554786)
- [Rin - Eientewi set 06 ~ Plain Asia](https://osu.ppy.sh/beatmapsets/993507) by [yaspo](https://osu.ppy.sh/users/4945926)
- [Rin - Lunatic set 16 ~ The Space Shrine Maiden Returns Home](https://osu.ppy.sh/beatmapsets/996256) by [defiance](https://osu.ppy.sh/users/6816873)
- [Rin - Mythic set ~ Last Occultism ~ Esotericist of the Present World](https://osu.ppy.sh/beatmapsets/1013620) by [Aeril](https://osu.ppy.sh/users/4334976)
- [Rin - Eientewi set 15 ~ Evening Primrose](https://osu.ppy.sh/beatmapsets/1014593) by [yaspo](https://osu.ppy.sh/users/4945926)
- [Rin - Kishinjou set 10 ~ Reverse Ideology](https://osu.ppy.sh/beatmapsets/1019506) by [defiance](https://osu.ppy.sh/users/6816873)

> ![Imperial Circus Dead Decadence header image](https://assets.ppy.sh/artists/35/header.jpg)

Reviewer 3: ```suggestion
![](https://assets.ppy.sh/artists/35/header.jpg)
```

The **ICDD: Kuruoshiku Saita Seisan na Mukuro wa Kanade, Itooshiku Saita Shoujo wa Seisen no Kotoba wo Utau** quest was completed by **[VINXIS](https://osu.ppy.sh/users/4323406)**, **[deetz](https://osu.ppy.sh/users/3513559)**, **[RLC](https://osu.ppy.sh/users/1047883)**, **[Nyquill](https://osu.ppy.sh/users/682935)**, **[Cheesecake](https://osu.ppy.sh/users/3700647)** and **[Sing](https://osu.ppy.sh/users/3795679)**.

- [Imperial Circus Dead Decadence - Uta](https://osu.ppy.sh/beatmapsets/962562) by [deetz](https://osu.ppy.sh/users/3513559)
- [Imperial Circus Dead Decadence - Ankoku No Jyoukaku Ni Shinkou Saru Igyou No Kyouki](https://osu.ppy.sh/beatmapsets/967766) by [Cheesecake](https://osu.ppy.sh/users/3700647)
- [Imperial Circus Dead Decadence - Tinu rareta Syuren Ni Fukeru Homura](https://osu.ppy.sh/beatmapsets/964134) by [VINXIS](https://osu.ppy.sh/users/4323406)
- [Imperial Circus Dead Decadence - Syakunetsu No Rengoku Ni Bara Wa...](https://osu.ppy.sh/beatmapsets/964680) by [Sing](https://osu.ppy.sh/users/3795679)

> ![Ben Briggs header image](https://assets.ppy.sh/artists/11/header.jpg)

Reviewer 3: ```suggestion
![](https://assets.ppy.sh/artists/11/header.jpg)
```

- [Ben Briggs - Fearless Living](https://osu.ppy.sh/beatmapsets/934415) by [Lasse](https://osu.ppy.sh/users/896613)
- [Ben Briggs - Yes (Maybe)](https://osu.ppy.sh/beatmapsets/933940) by [Myxo](https://osu.ppy.sh/users/2202645)
- [Ben Briggs - New Game Plus](https://osu.ppy.sh/beatmapsets/934627) by [Icekalt](https://osu.ppy.sh/users/5410645)
- [Ben Briggs - Wave Island](https://osu.ppy.sh/beatmapsets/934666) by [Gero](https://osu.ppy.sh/users/1467715)
- [Ben Briggs - Methods](https://osu.ppy.sh/beatmapsets/948165) by [LMT](https://osu.ppy.sh/users/7262798)
- [Ben Briggs - Selecting Continue](https://osu.ppy.sh/beatmapsets/948868) by [Mao](https://osu.ppy.sh/users/2204515)
- [Ben Briggs - Why Are We Yelling](https://osu.ppy.sh/beatmapsets/952380) by [Crissa](https://osu.ppy.sh/users/5405836)

> ![KIRA header image](https://assets.ppy.sh/artists/27/header.jpg)

Reviewer 3: ```suggestion
![](https://assets.ppy.sh/artists/27/header.jpg)
```

For this quest, the mappers had to **create and rank 5 mapsets of select vocaloid songs, each hosted by a different user**.

The **Vocaloid Showcase** quest was completed by **[Noffy](https://osu.ppy.sh/users/1541323)**, **[Kuron-kun](https://osu.ppy.sh/users/2697284)**, **[pishifat](https://osu.ppy.sh/users/3178418)**, **[Niva](https://osu.ppy.sh/users/197805)**, **[Seto Kousuke](https://osu.ppy.sh/users/2857314)**, **[hypercyte](https://osu.ppy.sh/users/9155377)** and **[bossandy](https://osu.ppy.sh/users/360437)**.

- [KIRA & Heartbreaker - B.B.F (feat. Hatsune Miku & Kagamine Rin)](https://osu.ppy.sh/beatmapsets/955866) by [hypercyte](https://osu.ppy.sh/users/9155377)
- [DJ'TEKINA//SOMETHING - Hidamari no Uta](https://osu.ppy.sh/beatmapsets/966324) by [Seto Kousuke](https://osu.ppy.sh/users/2857314)
- [Camellia - Kimi ga Shinu no o Nagamete Iru yo](https://osu.ppy.sh/beatmapsets/945476) by [pishifat](https://osu.ppy.sh/users/3178418)
- [KIRA - Magical Night (feat. Hatsune Miku)](http://osu.ppy.sh/beatmapsets/946771) by [Niva](https://osu.ppy.sh/users/197805)
- [baker - For a Dead Girl+](https://osu.ppy.sh/beatmapsets/955671) by [Noffy](https://osu.ppy.sh/users/1541323)

> ![sakuraburst header image](https://assets.ppy.sh/artists/8/header.jpg)

Reviewer 3: ```suggestion
![](https://assets.ppy.sh/artists/8/header.jpg)
```

The **Neuro Genre Showcase** quest was completed by **[Nifty](https://osu.ppy.sh/users/4956097)**, **[Cynplytholowazy](https://osu.ppy.sh/users/3901754)**, **[Faputa](https://osu.ppy.sh/users/845733)**, **[\_DUSK\_](https://osu.ppy.sh/users/6092181)**, **[Yales](https://osu.ppy.sh/users/2377881)**, **[Nepuri](https://osu.ppy.sh/users/6637817)**, **[Tyistiana](https://osu.ppy.sh/users/1421452)** and **[aabc271](https://osu.ppy.sh/users/155707)**

- [sakuraburst - cherry blossoms explode across the dying horizon](https://osu.ppy.sh/beatmapsets/878621) by [Cynplytholowazy](https://osu.ppy.sh/users/3901754)
- [Dictate - Cauldron](https://osu.ppy.sh/beatmapsets/942334) by [Nepuri](https://osu.ppy.sh/users/6637817)
- [sakuraburst - dragonlands](https://osu.ppy.sh/beatmapsets/935765) by [Faputa](https://osu.ppy.sh/users/845733)
- [Culprate & Joe Ford - Gaucho](https://osu.ppy.sh/beatmapsets/936234) by [Nifty](https://osu.ppy.sh/users/4956097)
- [sakuraburst - SELF DESTRUCT](https://osu.ppy.sh/beatmapsets/963193) by [\_DUSK\_](https://osu.ppy.sh/users/6092181)

> ![tieff header image](https://assets.ppy.sh/artists/34/header.jpg)

Reviewer 3: ```suggestion
![](https://assets.ppy.sh/artists/34/header.jpg)
```
